### ISSUE TITLE
method check in class Angle cannot be applied to given types

### ISSUE THREAD

**Issue by mibby** [CaptainObvious0/NoCheatPlus#6]
Compiling as of commit https://github.com/CaptainObvious0/NoCheatPlus/commit/fbd62a6f626f9f233ae141725eeb6af615dacd5c

@CaptainObvious0 Any ideas? Error on compilation attempt.
```
Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:2.5.1:compile (default-compile) on project ncpcore: Compilation failure
fr/neatmonster/nocheatplus/checks/fight/FightListener.java:[398,21] error: method check in class Angle cannot be applied to given types;
```

https://github.com/CaptainObvious0/NoCheatPlus/blob/325c3bad11769b77d11fd1105ce84ac61b39345e/NCPCore/src/main/java/fr/neatmonster/nocheatplus/checks/fight/FightListener.java#L398

**Comment by CaptainObvious0** [comment]
Should be fixed with ab5177e , I wasn’t able to compile and test it for myself, so let me know if that fixes it or not.

**Comment by mibby** [comment]
Confirmed compiles now, thanks!
